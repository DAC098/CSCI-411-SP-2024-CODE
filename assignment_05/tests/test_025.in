463 911
12
46
53
85
91
111
139
204
210
234
235
250
251
268
282
306
332
371
403
408
424
442
453
458
503
525
597
608
614
624
639
648
663
742
766
799
818
818
846
869
893
894
932
948
951
984
986
989
1004
1023
1050
1053
1059
1126
1126
1127
1198
1201
1204
1210
1212
1219
1222
1275
1298
1334
1363
1369
1414
1415
1454
1476
1486
1557
1568
1587
1594
1609
1621
1625
1638
1654
1664
1667
1669
1690
1698
1703
1836
1855
1864
1875
1914
1957
1988
2002
2006
2020
2031
2047
2114
2123
2140
2152
2164
2180
2198
2231
2234
2236
2310
2363
2365
2397
2413
2439
2473
2488
2526
2545
2577
2595
2599
2600
2603
2636
2643
2661
2683
2698
2715
2716
2723
2771
2773
2782
2790
2803
2808
2836
2844
2864
2921
2921
2924
2938
2947
2964
2972
2997
3022
3025
3025
3069
3072
3088
3092
3120
3182
3206
3238
3245
3246
3274
3277
3287
3304
3313
3345
3389
3410
3411
3417
3420
3428
3455
3463
3490
3497
3513
3524
3525
3535
3539
3626
3649
3653
3657
3699
3736
3749
3780
3791
3794
3805
3816
3820
3853
3858
3862
3867
3871
3887
3909
3920
3960
4009
4037
4048
4068
4104
4124
4189
4206
4232
4233
4234
4252
4285
4319
4362
4413
4459
4465
4473
4525
4531
4607
4622
4631
4661
4765
4831
4934
4939
4963
4965
4980
5008
5035
5083
5094
5127
5180
5217
5235
5237
5275
5284
5310
5366
5379
5382
5452
5473
5482
5494
5501
5550
5602
5683
5694
5716
5718
5726
5761
5784
5785
5800
5836
5845
5888
5903
5924
5939
6022
6024
6073
6089
6102
6110
6111
6130
6154
6157
6162
6167
6184
6195
6198
6232
6237
6266
6276
6283
6289
6293
6318
6352
6405
6445
6467
6509
6511
6511
6514
6537
6539
6551
6598
6610
6645
6661
6768
6770
6771
6781
6797
6840
6845
6871
6898
6900
7003
7034
7035
7049
7050
7101
7103
7112
7135
7140
7152
7178
7205
7291
7298
7300
7320
7355
7370
7418
7434
7442
7444
7459
7464
7479
7557
7579
7599
7606
7606
7617
7630
7847
7868
7873
7873
7893
7919
7926
7948
7977
7984
8007
8010
8047
8063
8095
8100
8158
8206
8214
8218
8225
8241
8273
8282
8283
8288
8323
8389
8403
8411
8414
8415
8422
8433
8501
8511
8575
8579
8637
8666
8686
8702
8727
8727
8737
8745
8778
8778
8790
8797
8799
8801
8810
8810
8830
8837
8863
8873
8907
8941
8994
9015
9032
9033
9048
9059
9062
9073
9114
9169
9186
9191
9286
9293
9310
9317
9332
9347
9350
9382
9404
9421
9472
9520
9541
9579
9602
9612
9636
9638
9674
9685
9696
9704
9722
9724
9729
9733
9751
9753
9762
9791
9856
9864
9866
9922
9977
