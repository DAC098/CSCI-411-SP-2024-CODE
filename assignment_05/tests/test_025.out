Enter the number of hotels and the ideal number of miles to travel per day: Enter 463 hotel distances each on a separate line: Hotels to visit: 43 90 135 189 228 258 300 339 374 424 463 
Total penalty: 3472
